From 1ea7b20e181365ba2b3b102f56ed0b90209afa2e Mon Sep 17 00:00:00 2001
From: Martin Kutlak <mkutlak@redhat.com>
Date: Wed, 25 Apr 2018 12:49:45 +0200
Subject: [PATCH] event: disable report_RHTSupport event and change URL for
 bugzilla

Set the URL for bugzilla to non-exist one because we do not want the centos
users report to bugzilla.

---
 src/plugins/bugzilla.conf         | 2 +-
 src/plugins/rhtsupport_event.conf | 8 ++++----
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/plugins/bugzilla.conf b/src/plugins/bugzilla.conf
index 51648de..159b188 100644
--- a/src/plugins/bugzilla.conf
+++ b/src/plugins/bugzilla.conf
@@ -1,5 +1,5 @@
 # Bugzilla URL
-BugzillaURL = https://bugzilla.redhat.com/
+BugzillaURL = https://bugzilla.example.com/
 # yes means that ssl certificates will be checked
 SSLVerify = yes
 # your login has to exist, if you don have any, please create one
diff --git a/src/plugins/rhtsupport_event.conf b/src/plugins/rhtsupport_event.conf
index cc2f05d..066f559 100644
--- a/src/plugins/rhtsupport_event.conf
+++ b/src/plugins/rhtsupport_event.conf
@@ -1,7 +1,7 @@
-EVENT=report_RHTSupport
+# EVENT=report_RHTSupport
     # Submit an uReport and create a case in Red Hat Customer Portal
-    reporter-rhtsupport -u
+    # reporter-rhtsupport -u
 
-EVENT=report_RHTSupport_AddData
+# EVENT=report_RHTSupport_AddData
     # Attach data to a case in Red Hat Customer Portal
-    reporter-rhtsupport -t
+    # reporter-rhtsupport -t
-- 
2.14.3

